assets/src/components/cd/services/ServicesColumns.tsx

```diff
@@ -114,7 +114,11 @@ export const ColRepo = columnHelper.accessor(({ node }) => node, {
           <ColWithIcon
             truncateLeft
             icon={
-              helm ? getProviderIconUrl('byok', theme.mode) : <GitHubLogoIcon />
+              svc?.helm ? (
```

Comment:
this is slightly wrong, we should really check if there's a helm url at all and show the helm-ish icon if so (you lose that info a bit by having one url var)

Reply:
I changed it to just check if it has a git repo instead, I assume it's one or the other right?

Reply:
Helm services can have git repos, so really the logic i gave is the best option

Reply:
ah makes sense. fixed